### DISCUSSION TITLE
Incorrect password

### DISCUSSION THREAD

**Discussion by Farman09** [nextgenhealthcare/connect#5648] [General]
Hi There,
Seems like I have no access to my Mirth account with the password I usually use. 
Is there a way to renew/update the password?

*Don't want to lose my already created channels.

Thanks in advance,

Farman

**Comment by pacmano1** [comment]
What Mirth account are you talking about?    If this is the login to the Mirth admin tool, there are already discussions here and on the forums about that.  What did you try?

**Comment by jonbartels** [comment]
https://github.com/nextgenhealthcare/connect/discussions/5421
